src/main/kotlin/no/nav/familie/ks/sak/kjerne/regelverk/Regelverk.kt

```diff
@@ -0,0 +1,6 @@
+package no.nav.familie.ks.sak.kjerne.regelverk
+
+enum class Regelverk {
+    FØR_LOVENDRING_2025,
+    LOVENDRING_FEBRUAR_2025,
```

Comment:
Kan fort tolke dette slik som at lovendringen kun gjelder for februar 2025.  Antar det er fra og med februar 2025? Prøvde på komme på et klokt navn men kom ikke på noe fornuftig selv 😅

Reply:
Vi gjør ingen endringer på navnene nå, siden faktisk dato osv kan endre seg siden ingenting er vedtatt. Har lagt inn en todo i koden på å oppdatere navnene når loven faktisk er vedtatt og vi vet mer om når lovendringen trer i kraf/om den har et navn